### ISSUE TITLE
Reference Error

### ISSUE THREAD

**Issue by rolfn** [zaygraveyard/moment-easter#2]
I think the line
```
if (module !== undefined) {
```
at end of "moment.easter.js" should be changed to
```
if (typeof module !== 'undefined') {
```
I get "ReferenceError: module is not defined" with the current definition.

**Comment by edbrannin** [comment]
(This would be fixed by PR #1)

**Comment by ziadkh0** [comment]
Closed via #1

**Comment by ziadkh0** [comment]
Published new version on npm
